bookmarks.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <h1>Quiz App</h1>
+    </header>
+    <nav>
+        <ul class="navbar">
+            <li><a href="index.html">🏠</a></li>
+            <li class="nav-active"><a href="bookmarks.html">💜</a></li>
+            <li><a href="profile.html">👤</a></li>
+        </ul>
+    </nav>
```

Comment:
my suggestion would be to define those icons in the footer of the webpage, but your solution looks fine as well